internal/planner/planner_test.go

```diff
@@ -37,9 +37,9 @@ func TestGeneratePlan(t *testing.T) {
 		expectUpdate                     int
 		expectWorkflow                   int
 		expectConfig                     bool
-		expectConfigSetCurrent           *bool    // pointer so we can test nil
-		expectConfigRampPercent          *float32 // pointer so we can test nil
-		maxVersionsIneligibleForDeletion *int32   // set by env if non-nil, else default 75
+		expectConfigSetCurrent           *bool  // pointer so we can test nil
+		expectConfigRampPercent          *int32 // pointer so we can test nil
```

Comment:
It looks like this is actually `expectConfigRampPercentBasisPoints`.
Ideally either rename the variable, or do the comparison so that the expectation can be in % instead of in basis points